packages/screens/ReadyLaunchpadApplications/ReadyLaunchpadApplicationsScreen.tsx

```diff
@@ -0,0 +1,332 @@
+import React, { ReactNode, useState } from "react";
+import { FlatList, StyleProp, TextStyle, View, ViewStyle } from "react-native";
+
+import avaPNG from "../../../assets/default-images/ava.png";
+import checkBadgeSVG from "../../../assets/icons/check-badge.svg";
+import SolanaCircleSVG from "../../../assets/icons/networks/solana-circle.svg";
+import { BrandText } from "../../components/BrandText";
+import { SVG } from "../../components/SVG";
+import { ScreenContainer } from "../../components/ScreenContainer";
+import { RoundedGradientImage } from "../../components/images/RoundedGradientImage";
+import { LinkView } from "../../components/linkView/LinkView";
+import { HighVolSortButton } from "../../components/sorts/HighVolSortButton";
+import { TableRow } from "../../components/table/TableRow";
+import { Tabs } from "../../components/tabs/Tabs";
+import { useIsMobile } from "../../hooks/useIsMobile";
+import { useAppNavigation } from "../../utils/navigation";
+import {
+  lightblue,
+  mineShaftColor,
+  neutral00,
+  neutral33,
+} from "../../utils/style/colors";
+import {
+  fontSemibold11,
+  fontSemibold13,
+  fontSemibold20,
+  fontSemibold28,
+} from "../../utils/style/fonts";
+import { layout, screenContentMaxWidthLarge } from "../../utils/style/layout";
+
+export type TabsListType = "readyForListing" | "waitingForApproval";
+
+const TABLE_ROWS = {
+  rank: {
+    label: "#",
+    flex: 1,
+  },
+  collectionNameData: {
+    label: "Collection Name",
+    flex: 5,
+  },
+  collectionNetwork: {
+    label: "Collection Network",
+    flex: 3,
+  },
+  projectReadinessForMint: {
+    label: "Project Readiness for Mint",
+    flex: 3,
+  },
+  whitelistQuantity: {
+    label: "Whitelist quantity",
+    flex: 2,
+  },
+  premiumMarketingPackage: {
+    label: "Premium marketing package",
+    flex: 2,
+  },
+  basicMarketingPackage: {
+    label: "Basic marketing package",
+    flex: 2,
+  },
+};
+
+const dummyData = {
+  rank: 1,
+  collectionNameData: "The R!ot",
+  collectionNetwork: "teritori",
+  projectReadinessForMint: "Complete and ready to mint",
+  whitelistQuantity: "0",
+  premiumMarketingPackage: "No",
+  basicMarketingPackage: "Yes",
+};
+
+export const ReadyLaunchpadApplicationsScreen: React.FC = () => {
+  const navigation = useAppNavigation();
+  const isMobile = useIsMobile();
+
+  const tabs = {
+    readyForListing: {
+      name: "Ready for listing",
+      badgeCount: 32,
+    },
+    waitingForApproval: {
+      name: "Waiting for approval",
+      badgeCount: 42,
+    },
+  };
+
+  const [selectedTab, setSelectedTab] =
+    useState<TabsListType>("readyForListing");
+
+  return (
+    <ScreenContainer
+      isLarge
+      footerChildren={<></>}
```

Comment:
Maybe remove the props if you don't put anything in it ? 👍

Reply:
this is the syntax to disable the footer, maybe we should change that